flink-ml-lib/src/main/java/org/apache/flink/ml/clustering/agglomerativeclustering/AgglomerativeClustering.java

```diff
@@ -70,6 +68,12 @@
  * second one contains the information of merging two clusters at each step. The data format of the
  * merging information is (clusterId1, clusterId2, distance, sizeOfMergedCluster).
  *
+ * <p>This operator supports the {@link org.apache.flink.ml.common.param.HasWindows} parameter,
+ * which creates mini-batches(windows) from input data and feeds each batch into this operator. The
+ * operator would perform the clustering operation on each mini-batch independently and output their
```

Comment:
Since the windowing strategy is used inside this operator rather than before this operator, it seems confusing to say `creates mini-batch ... and feeds each batch into this operator`.

How about the following doc:

This AlgoOperator splits input stream into mini-batches of elements according to the windowing strategy specified by the {@link org.apache.flink.ml.common.param.HasWindows} parameter, and performs the hierarchical clustering on each mini-batch independently. The clustering result of each element depends only on the elements in the same mini-batch.